holmes/config.py

```
@@ -649,20 +656,61 @@
-        model_params = {}
-        if self._model_list:
-            # get requested model or the first credentials if no model requested.
-            model_params = (
-                self._model_list.get(model_key, {}).copy()
-                if model_key
-                else next(iter(self._model_list.values())).copy()
+        selected_model: Optional[str] = None
+        selected_api_key: Optional[str] = (
+            self.api_key.get_secret_value() if self.api_key else None
+        )
+        selected_params: Dict[str, Any] = {}
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)